torch/onnx/symbolic_opset11.py

```diff
@@ -80,6 +80,13 @@ def index_put(g, self, indices_list_value, values, accumulate=False):
         ]
         index = g.op("Concat", *indices_list, axis_i=-1)
     else:
+        bool_inp = list(index.node().inputs())[0]
```

Comment:
Could you add a few comments maybe moving the ir snippet from the pass down to here? I'm confused why need to check input of index node.

Reply:
Moved ir snippet to opset files, I'm unsure how to structure the example

Reply:
do we need to copy masked_scatter comments as well?